docs/week3.md

```diff
@@ -3,34 +3,43 @@ This week, we’ll explore how cloud-based storage and machine learning platform
 
 ### Focus
 - Learn about different types of cloud storage.
+    - Databricks hive metastore
+    - Azure Blob Storage
 - Learn how cloud storage fits into ML workflows
 - Understand the differences between low-code and high-code approaches
 - Get hands-on experience with multiple Azure ML tools
+- Know how to visualize data with cloud tools
 
 ### Hands-On Activities
 - Upload an image dataset to Azure Blob Storage
 - Build a classifier using:
-  - Azure ML Designer (low-code drag-and-drop)
-  - Azure ML SDK (Python-based code-first)
+    - Azure ML Designer (low-code drag-and-drop)
+    - Azure ML SDK (Python-based code-first)
+    - Databricks AutoML
 - Compare pipeline transparency, usability, and deployment steps
+- Visualize features, accuracy, and other metrics with PowerBI on Azure
 - Run inference on new images in the cloud
 
 ### Learning Outcomes
 By the end of this week, students will be able to:
+
 - Explain the role of cloud storage in ML workflows
 - Upload and manage image data using Blob Storage
 - Build and deploy an ML model using Azure ML Designer
 - Use the Azure ML SDK to run inference in Python
 - Reflect on the trade-offs between low-code and code-first ML development:
-  - Usability
-  - Flexibility
-  - Cost and compute usage
+    - Usability
+    - Flexibility
+    - Cost and compute usage
+- Know how work with features and metrics with data visualization tools
```

Comment:
Yes good -- in the ML module we will go in detail over different metrics (accuracy, precision recall, F1), so we will want this ultimately in this lesson as well. Details will fill in as we build the actual lesson